godpi.go

```diff
@@ -18,8 +19,39 @@ var moduleList = []types.Module{
 }
 var cacheExpiration = 5 * time.Minute
 
+// Options allow end users init module with custom options
+type Options interface {
+	Apply(types.Module)
+}
+
+type MLOption struct {
+	TCPModelPath, UDPModelPath string
+	Threshold                  float32
+}
+
+func (o MLOption) Apply(mod types.Module) {
+	lsm := mod.(*ml.LinearSVCModule)
```

Comment:
use a type switch statement here

Reply:
```go
 lsm, ok := mod.(*ml.LinearSVCModule)
        if !ok {
                return
        }
```
type convert check, it that ok for you?

Reply:
That should be fine.

Reply:
Maybe we could move this option to a new file however in the same package (options.go?) to keep the module-specific logic outside godpi.go.

Reply:
Sure